docs/nx/azure-react/generator-app-insights-web.md

```diff
@@ -0,0 +1,102 @@
+### @ensono-stacks/azure-react:app-insights-web
+
+<details>
+<summary>Installs and configures a ReactJS app with App Insights.</summary>
+
+This generator will add and configure [applicationinsights reactjs](https://www.npmjs.com/package/@microsoft/applicationinsights-react-js) and [applicationinsights web](https://www.npmjs.com/package/@microsoft/applicationinsights-web) npm packages for you.
+
+This enables the following:
+
+- Tracking of router changes
+- React components usage statistics
+
+It requires the connection string to be set in an env variable, from which it will be read.
+
+## Usage
+
+```bash
+nx generate @ensono-stacks/azure-react:app-insights-web --name=<app-name> --connectionString=<KEY_NAME> --directory=<directory where the lib is placed> --importPath=<import path for the library> --tags=<tags to the library>
```

Comment:
Don't think we need to list all cli arguments here in addition to the table below. That's assuming the plugin prompts with questions to gather information for the required values. (Makes it nice and easy to just hit copy/paste then)
```suggestion
nx generate @ensono-stacks/azure-react:app-insights-web
```